tests/risk/test_risk_period.py

```diff
@@ -385,7 +385,7 @@ def test_benchmarkrange(self):
     def test_partial_month(self):
 
         start_session = self.trading_calendar.minute_to_session_label(
-            pd.Timestamp("1991-01-01", tz='UTC')
+            pd.Timestamp("1993-02-01", tz='UTC')
```

Comment:
It does not. I have to change that comment as I had misunderstood how google provides data. The one important restriction is that we only can get the last 4000 days of data at a time.

I stitched multiple csvs of data all the way back to 1993, the furthest possible date google provides data for.